_po/ja/how-to/laravel.po

```diff
@@ -12,53 +12,71 @@ msgid ""
 "title: How to use PGroonga with Laravel\n"
 "---"
 msgstr ""
+"---\n"
+"title: Laravelと一緒に使う方法\n"
+"---"
 
 msgid ""
 "# How to use PGroonga with Laravel\n"
 "In this guide we will walk through building a Laravel application from scratch"
 " with PGroonga."
 msgstr ""
+"# Laravelと一緒に使う方法\n"
+"ここではLaravelでPGroongaを使うアプリを一から作っていきます。"
 
 msgid "## Installation"
-msgstr ""
+msgstr "## インストール"
 
 msgid ""
 "### Installing Laravel\n"
 "**NOTE**: This tutorial assumes you have already set up your environment for L"
 "aravel, PostgreSQL and PGroonga"
 msgstr ""
+"### Laravelのインストール\n"
+"**注意**: このチュートリアルは既にLaravelとPostgreSQLとPGroongaが使える前提となります"
 
 msgid ""
 "If you have already installed PHP and Composer on your local machine, you may "
 "create a new Laravel project via Composer:"
 msgstr ""
+"PHPとComposerがインストール済みの場合は、Composerを使い新規のLaravelプロジェクトを作ります:"
 
 msgid ""
 "```shell\n"
 "composer create-project laravel/laravel pgroonga_laravel\n"
 "```"
 msgstr ""
+"```shell\n"
+"composer create-project laravel/laravel pgroonga_laravel\n"
+"```"
 
 msgid ""
 "After the project has been created, start Laravel’s local development server u"
 "sing the Laravel’s Artisan CLI serve command:"
 msgstr ""
+"プロジェクト生成後は下記のArtisan CLIコマンドでローカル環境のLaravelが立ち上げ可能です:"
 
 msgid ""
 "```shell\n"
 "cd pgroonga_laravel\n"
 "php artisan serve\n"
 "```"
 msgstr ""
+"```shell\n"
+"cd pgroonga_laravel\n"
+"php artisan serve\n"
+"```"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.